lib/symbConstant.ml

```diff
@@ -77,11 +87,17 @@ module Make(Scalar:Scalar.S) = struct
       Misc.string_eq  s1 s2 && Misc.int_eq p1 p2
   | Tag t1,Tag t2 -> Misc.string_eq t1 t2
   | PteVal p1,PteVal p2 -> PTEVal.compare p1 p2 = 0
-  | (PteVal _,(Symbolic _|Concrete _|Label _|Tag _))
-  | (Concrete _,(Symbolic _|Label _|Tag _|PteVal _))
-  | (Symbolic _,(Concrete _|Label _|Tag _|PteVal _))
-  | (Label _,(Concrete _|Symbolic _|Tag _|PteVal _))
-  | (Tag _,(Concrete _|Symbolic _|Label _|PteVal _))
+  | MemAttr l1,MemAttr l2 ->
```

Comment:
Is there some assumed sorting of attributes?
If not `["A";"B"]`  and `["B";"A";] are different, if yes they are identical and the above comparison function is wrong.

In any case an appropriate comment near the definition of `MemAttr` would be useful.